ktor-http/common/src/io/ktor/http/Cookie.kt

```diff
@@ -133,22 +133,10 @@ public fun renderSetCookieHeader(cookie: Cookie): String = with(cookie) {
 }
 
 /**
- * Format `Set-Cookie` header value
+ * Format `Cookie` header value
  */
 public fun renderCookieHeader(cookie: Cookie): String = with(cookie) {
```

Comment:
This function seems only being used by https://github.com/ktorio/ktor/blob/14c687426471ea854a4cf96b4c91de38ebf4fd37/ktor-client/ktor-client-core/common/src/io/ktor/client/request/utils.kt#L34-L47
But the implementation was not appropriate; it didn't conform to `Cookie` header spec in [MDN doc](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cookie) and [RFC6265 spec](https://httpwg.org/specs/rfc6265.html#cookie) by potentially containing attribute pairs such as `Max-Age=...; Expires=...;`. This could causes a problem when trying to parse cookies from such string, as `Max-Age`, `Expires` will be treated as cookies themselves.